scripts/build.sh

```diff
@@ -26,10 +26,10 @@ else
 fi
 
 echo $BUILD_VERSION >$VERSION_FILE
-
-GOGC=1500 go install \
+GOGC=1500 CGO_ENABLED=0 go install \
+	-a -installsuffix cgo \
 	-ldflags "-X $PKG_NAME.version=$BUILD_VERSION \
 	-X $PKG_NAME.buildTime=$BUILD_TIME \
 	-X $PKG_NAME.gitCommit=$GIT_COMMIT \
```

Comment:
Why do you need to disable CGO ?

Reply:
@rchirakk: This is needed to build static binaries which don't rely on glibc at all. We need static binaries which don't use cgo and any kind of features which require libc functions. This is something others have done as well.

Reply:
We do this on auth_proxy as well